internal/engine/wazevo/backend/isa/amd64/instr_encoding_test.go

```diff
@@ -3895,19 +3895,89 @@ func TestInstruction_format_encode(t *testing.T) {
 			wantFormat: "exit_sequence %r15",
 		},
 		{
-			setup:      func(i *instruction) { i.asXCHG(r11VReg, r14VReg) },
+			setup:      func(i *instruction) { i.asXCHG(r11VReg, newOperandReg(r14VReg), 8) },
 			want:       "4d87f3",
-			wantFormat: "xchg %r11, %r14",
+			wantFormat: "xchg.q %r11, %r14",
```

Comment:
JFYI I noticed the register order is reversed from capstone

Reply:
good catch!

Reply:
Oh thought just a formatting issue 😂